pages/tech-apps-accelerated-mobile-pages.js

```diff
@@ -1,88 +1,118 @@
-import React from 'react';
-import PropTypes from 'prop-types';
+import React from "react";
+import PropTypes from "prop-types";
 
-import Layout from '../components/layout/main';
-import ContentNavigation from '../components/content/navigation';
+import Layout from "../components/layout/main";
+import ContentNavigation from "../components/content/navigation";
 
-import { techApps } from '../main.config';
+import { techApps } from "../main.config";
```

Comment:
@antonmorya , why are you changing single quotes to double ones? Please leave single ones.